src/components/CommentList/CommentList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Comment } from '../Comment'
+import { commentType } from '../Comment'
+
+export const CommentList = ({allComments}) => (
```

Comment:
why allComments?
just comments
```suggestion
export const CommentList = ({ comments }) => (
```

please, fix linter errors